packages/components/src/components/popover/popover.tsx

```diff
@@ -52,47 +55,65 @@ export class Popover {
   private popover: HTMLDivElement;
   private button: HTMLButtonElement;
 
+  private isNative: boolean = false;
+  private table: HTMLElement;
+
   public connectedCallback(): void {
     addDocumentEventListener(this);
+    this.detectNativeCase();
   }
 
   public componentShouldUpdate(newVal: unknown, oldVal: unknown): boolean {
     return hasPropValueChanged(newVal, oldVal);
   }
 
   public componentDidRender(): void {
-    if (this.open) {
-      // calculate / update position only possible after render
-      updatePopoverStyles(this.host, this.spacer, this.popover, this.direction, this.theme);
+    if (this.isNative && this.spacer?.matches(':popover-open')) {
+      addNativeScrollListeners(this.host, this.table, this.spacer);
+      // Set new popover position depending on button position
+      updateNativePopoverStyles(this.spacer, this.button);
+      // Update popover styles with new position
+      updatePopoverStyles(this.host, this.spacer, this.popover, this.direction, this.theme, this.isNative);
+    } else {
+      if (this.open) {
+        // calculate / update position only possible after render
+        updatePopoverStyles(this.host, this.spacer, this.popover, this.direction, this.theme);
```

Comment:
Why does only one the 2nd need the `this.open` condition?

Reply:
Since the the native popover doesn't rely on this.open